test/clients/azure/client.go

```diff
@@ -1,30 +1,52 @@
 package azure
 
 import (
+	"context"
 	"os"
 
 	"github.com/openshift/openshift-azure/pkg/util/azureclient"
+	"github.com/openshift/openshift-azure/pkg/util/azureclient/storage"
+	"github.com/openshift/openshift-azure/pkg/util/cloudprovider"
+	"github.com/openshift/openshift-azure/pkg/util/configblob"
 )
 
 type Client struct {
 	Accounts                         azureclient.AccountsClient
 	Applications                     azureclient.ApplicationsClient
+	BlobStorage                      storage.BlobStorageClient
 	VirtualMachineScaleSets          azureclient.VirtualMachineScaleSetsClient
 	VirtualMachineScaleSetExtensions azureclient.VirtualMachineScaleSetExtensionsClient
 	VirtualMachineScaleSetVMs        azureclient.VirtualMachineScaleSetVMsClient
 }
 
-func NewClientFromEnvironment() (*Client, error) {
+func NewClientFromEnvironment(setStorageClient bool) (*Client, error) {
```

Comment:
why do you need to conditionally set the storageClient?

Reply:
Because it makes API calls and most suites don't need to set it.